core/domain/feedback_services.py

```diff
@@ -179,7 +182,7 @@ def get_last_updated_time(exploration_id):
 
     If this exploration has no threads, returns None.
     """
-    threadlist = get_all_threads(exploration_id, False)
+    threadlist = get_displayable_threads(exploration_id, False)
     return max(
```

Comment:
By grepping "feedback_last_updated" couldn't find any front-end code using it as a url. 
So removing it.